jui-core/index.d.ts

```diff
@@ -33,9 +37,8 @@ interface UtilBase {
      * @param {Function} ctor base Class
      * @param {Function} superCtor super Class
      */
-    /* tslint:disable-next-line:callable-types */
-    inherit (ctor: () => void, superCtor: () => void): void;
-    /* tslint:enable-next-line:callable-types */
+    inherit ( ctor: ClassFunction, superCtor: ClassFunction ): void;
```

Comment:
This will result in lint errors due to extra spaces. Try `inherit(ctor: ClassFunciton, superCtor: ClassFunction): void;`.